exercises/practice/rectangles/.meta/example.ua

```diff
@@ -0,0 +1,12 @@
+ValidPair  ← /×≡(=°⊟) ⊏
+ValidXs    ← ValidPair [0_1 2_3]
+ValidYs    ← ValidPair [0_2 1_3]
+Valid      ← ×⊃(ValidXs≡⊢|ValidYs≡⊣)
+Parse      ← ⊜∘(≠@\n.)
+Candidates ← ▽≡Valid.⧅<4⊚⌕@+
+Rectangle ← (
+  ∩(⍜-⇡+1⊃(/↥|/↧))≡(⊃⊢⊣)
+  ◴⊂⊂⊃(⊂⊃(⊞⊂⊢|⊞⊂⊣)|≡⇌⊃(⊞⊂⊢|⊞⊂⊣)⊂⊃(⊢|⊣):)
+  /↧≠@  ≡⊡ ⊙¤
+)
+Rectangles ← /+≡Rectangle⊙¤⊸Candidates Parse
```

Comment:
This doesn't really work. I'm putting this PR up in the hope that someone can solve this more easily than I can :)